src/Main.java

```diff
@@ -1,5 +1,221 @@
+                case 1:
+                    adminMenu();
+                    break;
+                case 2:
+                    userMenu();
+                    break;
+                case 3:
+                    running = false;
+                    System.out.println("Exiting... Goodbye!");
+                    break;
+                default:
+                    System.out.println("Invalid option. Try again.");
+            }
+        }
+    }
+
+    private static void printMainMenu() {
+        System.out.println("\nWelcome to Coworking Space Reservation System");
+        System.out.println("1. Admin Login");
+        System.out.println("2. User Login");
+        System.out.println("3. Exit");
+        System.out.print("Choose an option: ");
+    }
+
+    private static int getUserInput() {
+        while (!scanner.hasNextInt()) {
+            System.out.print("Invalid input. Enter a number: ");
+            scanner.next();
+        }
+        return scanner.nextInt();
+    }
+
+    private static void adminMenu() {
+        System.out.println("\nAdmin Menu");
+        System.out.println("1. Add a new coworking space");
+        System.out.println("2. Remove a coworking space");
+        System.out.println("3. View all reservations");
+        System.out.println("4. Back to Main Menu");
+        System.out.print("Choose an option: ");
+
+        int choice = getUserInput();
+
+        switch (choice) {
+            case 1:
+                addWorkspace();
+                break;
+            case 2:
+                removeWorkspace();
+                break;
+            case 3:
+                viewAllReservations();
+                break;
+            case 4:
+                System.out.println("Returning to Main Menu...");
+                break;
+            default:
+                System.out.println("Invalid option. Try again.");
+        }
+    }
+
+    private static void userMenu() {
+        System.out.println("\nCustomer Menu");
+        System.out.println("1. Browse available spaces");
+        System.out.println("2. Make a reservation");
+        System.out.println("3. View my reservations");
+        System.out.println("4. Cancel a reservation");
+        System.out.println("5. Back to Main Menu");
+        System.out.print("Choose an option: ");
+
+        int choice = getUserInput();
+
+        switch (choice) {
+            case 1:
+                browseSpaces();
+                break;
+            case 2:
+                makeReservation();
+                break;
+            case 3:
+                viewUserReservations();
+                break;
+            case 4:
+                cancelReservation();
+                break;
+            case 5:
+                System.out.println("Returning to Main Menu...");
+                break;
+            default:
+                System.out.println("Invalid option. Try again.");
+        }
+    }
+
+    private static void addWorkspace() {
```

Comment:
avoid using static methods everywhere and follow Single responsibility principle (ex. Create AdminService for the admin actions and use instance of the class)